mysql-test/main/func_json.result

```diff
@@ -785,13 +785,13 @@ drop table t1;
 #
 SELECT JSON_SET('{}', '$.a', _utf8 0xC3B6);
 JSON_SET('{}', '$.a', _utf8 0xC3B6)
-{"a": "�"}
+{"a": "�"}
 SELECT JSON_SET('{}', '$.a', _utf8 0xC3B6, '$.b', _utf8 0xC3B6);
 JSON_SET('{}', '$.a', _utf8 0xC3B6, '$.b', _utf8 0xC3B6)
-{"a": "�", "b": "�"}
+{"a": "�", "b": "�"}
 SELECT JSON_SET('{}', '$.a', _utf8 X'C3B6', '$.x', 1, '$.b', _utf8 X'C3B6');
 JSON_SET('{}', '$.a', _utf8 X'C3B6', '$.x', 1, '$.b', _utf8 X'C3B6')
-{"a": "�", "x": 1, "b": "�"}
+{"a": "�", "x": 1, "b": "�"}
 #
```

Comment:
It looks like you editor is turning some letters into weird �   Everything else looks ok.

Reply:
fixed.